Training/Easy/power_of_thor.rb

```diff
@@ -7,58 +7,52 @@ def initialize
     @light_x, @light_y, @x, @y = gets.split(' ').collect(&:to_i)
   end
 
-  def find_direction
-    direction = ''
-    if x < light_x
-      direction = 'E'
-      if y > light_y
-        direction = 'NE'
-      else
-        direction = 'SE' unless y == light_y
-      end
-    elsif x == light_x
-      if y > light_y
-        direction = 'N'
-      else
-        direction = 'S'
-      end
+  def move_north
+    self.y -= 1
+    if x == light_x
+      puts 'N'
```

Comment:
Take the `puts` statements out of all of the `move_` methods - make the methods return the string value, and then use `puts` inside the `loop`. The reason is that if you change the way you output (in this case, `puts`), you will have to refactor all of the `move_` methods, as opposed to just changing it in one place.